### PR TITLE
ath79: add support for TP-Link Archer C2 V3 (AC900) [v2]

### PR DESCRIPTION
**Pull request by lars18th**
This "new" PR only resolves the conflict pending in the PR #1501 .
The original work is from @x-logLT.

### CODE REVIEW

**Comment by lars18th** [comment]
Hi @mkresin & @x-logLT ,

No comments?
Still waiting for it to be merged!

**Comment by lars18th** [comment]
> I think the reason why nobody is looking at the PR is because you need to use your real name and email address in the patches/commits as per the [OpenWrt Guidelines](https://openwrt.org/submitting-patches).
> Also don't forget: you'll also need to add your Signed-off-by: tag to the patches, even if the are not your own in case you want to follow through with the PR.

Hi @chunkeey ,

Sorry, but I can't do that! :cry: 

However, you, @mkresin or the original author (@x-logLT ) can sign the patch.
So, I suggest to review the patch, and create a new [v3] signed. Then remove [v1] and [v2] and merge it.

You agree?

**Comment by chunkeey** [comment]
> Hi @chunkeey ,
> 
> Sorry, but I can't do that! cry
> 
> However, you, @mkresin or the original author (@x-logLT ) can sign the patch.
> So, I suggest to review the patch, and create a new [v3] signed. Then remove [v1] and [v2] and merge it.
> 
> You agree?

The "Signed-off-by:" tag and the real name is really important. If you are unaware of the history and the place it has you could read up on it in this [stackoverflow thread](https://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for) of why it was introduced in the linux kernel and what it means for Projects like OpenWrt and your PR.

If you can't provide your real name for the "Signed-off-by" tags, please close the PR. Thanks.

File: target/linux/ath79/dts/qca9563_tplink_archer-c2-v3.dts [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+				reg = <0x000000 0x020000>;
+				read-only;
+			};
+
+			partition@20000 {
+				label = "uboot";
+				reg = <0x020000 0x10000>;
+			};
+			
+			partition@30000 {
```
Comment (rmilecki) [inline target/linux/ath79/dts/qca9563_tplink_archer-c2-v3.dts]: Please add `compatible`, see
[OpenWrt-Devel] Specifying "firmware" partition format on all DTS targets
http://lists.infradead.org/pipermail/openwrt-devel/2018-November/014706.html
for details.

**Comment by lars18th** [comment]
> If you can't provide your real name for the "Signed-off-by" tags, please close the PR. Thanks.

Done!

**Pull request closed without merging**